src/lib/Preview.js

```diff
@@ -814,6 +814,11 @@ class Preview extends EventEmitter {
      * @return {void}
      */
     setupUI() {
+        // Do nothing if container is already setup and in the middle of retrying
+        if (this.container && this.retryCount > 0) {
```

Comment:
It doesn't look like `this.container` gets cleared out in the destroy or navigate, so this may cause issues

Reply:
created the method `PreviewUI.isSetup()` instead